utils.go

```diff
@@ -104,10 +105,10 @@ func formatError(err error) error {
 		default:
 			return fmt.Errorf("%w, %v", services.ErrUnexpected, err)
 		}
-	case fn("40400001"):
+	case fn(strconv.Itoa(responseCodeFileOrDirectoryNotFound)):
```

Comment:
Can we also can the code like we does in `checkErrorCode`

Reply:
I‘ve updated as my understanding. Please help to review.